src/daemon/frame.h

```diff
@@ -99,6 +99,11 @@ static union {
 #define PEEK_RESTORE POKE_RESTORE
 
 /* LLDP specific. We need a `tlv' pointer. */
+#define POKE_START_LLDP_TLV_8023BT(type)  \
+        (			   \
+	 tlv = pos,		   \
+	 POKE_UINT16((type << 9) | (0x1d))	   \
+	)
```

Comment:
It's only used once, it doesn't seem worth it.

Reply:
Is your suggestion that the contents of the macro should be done in line in lldp.c instead?

Reply:
Yes.